student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
```

Comment:
Please use `require_relative` here to load a local file as `require` is used to load installed gems.